ibis/bigquery/compiler.py

```diff
@@ -334,6 +331,7 @@ def truncator(translator, expr):
             )
 
         return "{}_TRUNC({}, {})".format(kind, arg, unit)
+
```

Comment:
Generally speaking, remove the whitespace changes. There was only a single whitespace lint error in CI and it was around the new `@compiles`/`@rewrites` functions that you introduced